README.md

```diff
@@ -55,9 +55,9 @@ This is the common workflow to build and deploy locally:
   make info-swarm
 
   # open front-end in the browser
-  #  localhost:9081 - simcore front-end site
+  #  127.0.0.1.nip.io:9081 - simcore front-end site
   #
-  xdg-open http://localhost:9081/
+  xdg-open http://127.0.0.1.nip.io:9081/
```

Comment:
I think adding something like in the 

[nip.io](https://nip.io/) is a service that maps ``<anything>[.-]<IP Address>.nip.io`` in "dot", "dash" or "hexadecimal" notation to the corresponding ``<IP Address>``

But again, I am not sure what value this adds in this case vs what we had before. Could you clarify?

Reply:
Will update the docs to make them look cleaner.

🤦 I forgot to mention the most important fact. Because the `dynamic-sidecar` is exposed on `UUID.services.SOME_DNS` it is required to use a DNS otherwise the dynamic sidecar will not be reached by the browser.
This is a side effect of how Traefik is configured and **more importantly** because `UUID.services.127.0.0.1` **is not a valid DNS**.
Meanwhile `UUID.services.127.0.0.1.nip.io` is totally fine.

It might be obvious, but this change will only impact the development cycle.

Reply:
@pcrespov @sanderegg the docs were updated (especially the Makefile) I would not add the information here.